app/views/pages/index.html.haml

```diff
@@ -46,7 +47,6 @@
       - elsif Rails.configuration.hackathon['registration_is_open']
         %div
           = btn_link_to "Register Now", new_user_registration_path
-          = btn_link_to "Volunteer", volunteer_link
```

Comment:
nit: probably want to keep the volunteer link here for the future

Reply:
I just commented it out for the future